week1/react-exercises/hobbies/src/components/HobbyList.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import Hobby from "./Hobby";
+
+const HobbyList = ({ data }) => {
+  // using index is not considered good practice
+  const hobbyComponents = data.map((el, i) => <Hobby key={i} hobby={el} />);
+  return <ul className="hobby-list">{hobbyComponents}</ul>;
```

Comment:
There is no mistake here but we generally prefer to place all the JSX after the return statement and not on the function body
```suggestion
  return <ul className="hobby-list">{data.map((el, i) => <Hobby key={i} hobby={el} />)}</ul>;
```